htdocs/web_portal/views/site/view_site.php

```diff
@@ -50,7 +50,7 @@
         <!--  Contacts -->
         <div class="tableContainer" style="width: 55%; float: left;" >
             <span class="header" style="vertical-align:middle; float: left; padding-top: 0.9em; padding-left: 1em;">Contact Info</span>
-            <img src="<?php echo \GocContextPath::getPath() ?>img/contact_card.png" height="25px" style="float: right; padding-right: 1em; padding-top: 0.5em; padding-bottom: 0.5em;" />
+            <img src="<?php echo \GocContextPath::getPath() ?>img/contact_card.png" height="32px" style="float: right; padding-right: 1em; padding-top: 0.5em; padding-bottom: 0.5em;" />
```

Comment:
That looks better. I think the one tweak left for these icons would be to remove `padding-bottom` which will make the icon appear about the same size as the `my_sites.php` ones and put it a bit more in the centre vertically. Also has the advantage of removing some inline CSS!

Reply:
Removed the inline CSS by using the decoration definition in order to have consitancy with the other icons

Reply:
Nice.